edbterraform/data/terraform/aws/modules/database/outputs.tf

```diff
@@ -45,3 +45,7 @@ output "tags" {
 output "resource_id" {
   value = aws_db_instance.rds_server.identifier
 }
+
+output "instance_id" {
+  value = aws_db_instance.rds_server.identifier
+}
```

Comment:
Is this supposed to be `id`?

Reply:
Looks like I will have to retest, because the resource_id is not up to date in my edb-benchmarks repo.

In the benchmark, `resource_id `is used for getting the metrics from AWS. But this would not let me gather postgres error logs. Instead, the `instance_id` was needed.

Reply:
I think this branch can be deleted. After refreshing my local terraform and testing, I believe the only value we need is this one:
```
output "resource_id" {
  value = aws_db_instance.rds_server.identifier
}
```